src/test/groovy/org/gradle/android/TestVersions.groovy

```diff
@@ -42,6 +42,8 @@ class TestVersions {
 
     static List<String> supportedKotlinVersions = ["1.6.21", "1.7.21", "1.8.0"]
 
+    static List<String> supportedKspVersions = ["1.7.22-1.0.8", "1.8.10-1.0.9"]
```

Comment:
should we align the versions with the kotiln versions here?

Reply:
done https://github.com/gradle/android-cache-fix-gradle-plugin/pull/429/commits/250eb1a2f828caf68e6a26894b286e1d0be49053

created a map associating Kotlin versions with Ksp

Reply:
nice